apps/admin-gui/src/app/shared/components/consent-hubs-list/consent-hubs-list.component.ts

```diff
@@ -33,8 +34,11 @@ export class ConsentHubsListComponent implements OnChanges {
 
   @Input() consentHubs: ConsentHub[];
   @Input() filterValue = '';
-  @Input() displayedColumns: string[] = ['id', 'name', 'enforceConsents', 'facilities'];
+  @Input() displayedColumns: string[] = ['select', 'id', 'name', 'enforceConsents', 'facilities'];
   @Input() tableId: string;
+  @Input() selection = new SelectionModel<ConsentHub>(true, []);
+
+  @Output() selectionChanged: EventEmitter<void> = new EventEmitter<void>();
```

Comment:
please look on how we are doing selection in other tables, I don't think that the selection changed is needed (eg. members-list)

Reply:
Okay, I will remove it. I have seen it in services-list and maybe some others, that's why it's here as well.